packaging/nnstreamer.spec

```diff
@@ -381,25 +381,23 @@ cp -r result %{buildroot}%{_datadir}/nnstreamer/unittest/
 %files -n capi-nnstreamer
 %manifest capi-nnstreamer.manifest
 %license LICENSE
-%{_libdir}/libcapi-nnstreamer.so.*
+%{_libdir}/libcapi-nnstreamer.so
```

Comment:
What about adding symlinks of libcapi-nnstreamer.so.0 for the compatibility against Tizen-FX C# files?

Reply:
ACK!